tests/homePage/homePageHeader2.spec.js

```diff
@@ -0,0 +1,106 @@
+import { test, expect } from '@playwright/test';
+
+const HomePageUrl = 'https://magento.softwaretestingboard.com/';
+
+test('Search suggestion in header', async ({ page }) => {
+  const inputSelector = '#search';
+
+  await page.goto(HomePageUrl);
+  await expect(page).toHaveTitle('Home Page');
+
+  await page
+    .locator(inputSelector)
+    .pressSequentially('jackets', { delay: 100 });
+  await Promise.resolve(
+    page.waitForResponse(
+      (resp) =>
+        resp.url().includes('/search/ajax/suggest') && resp.status() === 200
+    )
+  );
+
+  const parentLocator = page.locator('#search_autocomplete');
+  const childLocator = parentLocator.locator('[role="option"]');
+
+  const lenghtEl = await childLocator.count();
+  expect(lenghtEl).toEqual(8);
+
+  for (let i = 0; i < lenghtEl; i++) {
+    await expect(childLocator.nth(i)).toContainText('jacket', {
+      ignoreCase: true,
+    });
+  }
+});
+
+test('Logo click redirect home page', async ({ page }) => {
+  await page.goto(HomePageUrl);
+  const logo = page.getByLabel('store logo');
+  await logo.click();
+  await page.waitForURL(HomePageUrl);
+  expect(page.url()).toEqual(HomePageUrl);
```

Comment:
Last line you forgot an await, and actually that line was executed before the other ones. So if you would have had a different URL this would have failed. Also no need to put stuff in constants if you use only once

```suggestion
  await page.goto(HomePageUrl);
  await page.getByLabel("store logo").click();
  await expect(page.url()).toEqual(HomePageUrl);
```